.github/scripts/download-bundled/pcre2-adjust-sljit-directory.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ext/pcre/pcre2lib/pcre2_jit_compile.c b/ext/pcre/pcre2lib/pcre2_jit_compile.c
```

Comment:
We have some existing patch files, e.g. ext/fileinfo/libmagic.patch. They should live in a consistant place.

Reply:
Yes, to be consistent.

I personally prefer to keep it here for now as there are dozens of bundled sources and maintaining the update files is much easier if they are kept at shared place.